app/controllers/ctc/questions/dependents/child_lived_with_you_controller.rb

```diff
@@ -6,9 +6,9 @@ class ChildLivedWithYouController < BaseDependentController
         layout "yes_no_question"
 
         def self.show?(dependent)
-          return false unless dependent&.relationship
+          return false unless dependent.present? && dependent.relationship.present?
 
-          dependent.qualifying_child_relationship? && dependent.yr_2020_meets_qc_age_condition? && dependent.meets_qc_misc_conditions? && !dependent.yr_2020_born_in_final_6_months?
+          return true unless dependent.born_in_final_6_months_of_tax_year?(TaxReturn.current_tax_year)
```

Comment:
temporarily removing some of the logic for show here since the later questions affect newer questions